### DISCUSSION TITLE
Provider With Easy to Use Port Forwarding

### DISCUSSION THREAD

**Discussion by g8tor888** [haugene/vpn-configs-contrib#251] [General]
I switched to ProtonVPN becuase I saw they support port forwarding. Its turning out to be a pain in the ass to setup. This is the closest I've seen ( https://github.com/haugene/docker-transmission-openvpn/discussions/2633 ) seems brittle if I can even get it running. I'd like to switch to a provider that's easier to use port forwarding with. Can anyone make any recommendations?

Thanks!

**Comment by pkishino** [comment]
I have been using PrivateVPN for years now after nordvpn turned to shit.. works great except the WireGuard doesn’t do port forwarding currently, but openvpn works well

**Reply by g8tor888** [reply]
Thanks for the reply. How do you setup port forwarding with PrivateVPN for the transmission container?
